packages/replay/src/replay.ts

```diff
@@ -1056,8 +1056,9 @@ export class ReplayContainer implements ReplayContainerInterface {
   private _onMutationHandler = (mutations: unknown[]): boolean => {
     const count = mutations.length;
 
-    const mutationLimit = this._options._experiments.mutationLimit || 0;
-    const mutationBreadcrumbLimit = this._options._experiments.mutationBreadcrumbLimit || 1000;
+    const mutationLimit = this._options.mutationLimit || this._options._experiments.mutationLimit || 0;
```

Comment:
Do we need to keep this backwards compatible with the experiments? 🤔 esp. as we also changed the semantic of the mutation limit here, I would just remove the experiment and switch this to the new syntax. we can/should call this out properly in the changelog then?

Reply:
I'm good with that